### DISCUSSION TITLE
Help a begginer to start using

### DISCUSSION THREAD

**Discussion by fpoliva** [JSBSim-Team/jsbsim#501] [Q&A]
Hello everybody,

My name is Flavio and I am a Brazilian PhD student.

I'm literally starting to use JSBSim, but I'm a little lost on how to run simulations.

I intend to use it as the main simulation tool in my research, because the connection to FlightGear would help to improve the presentation of the results.

I was able to successfully perform the installation and tests proposed in the online manual (https://jsbsim-team.github.io/jsbsim-reference-manual/mypages/quickstart/).

My initial goal is to use an existing aircraft model to learn how to trim the aircraft and then run maneuver simulations (e.g. pushover, pull-ups, windup turns, and so on).
Next, I intend to implement a model of my study aircraft and perform the necessary analyses.

Do you have any material or can help me in any other way to take the next steps?

Thank you very much,
Flavio.

**Comment by seanmcleod** [comment]
Hi Flavio. You have 2 main options, you can make use of the scripting facility or you can make use of the python api.

I'd suggest reading through the old reference manual - http://jsbsim.sourceforge.net/JSBSimReferenceManual.pdf

And then looking at the sample scripts in the scripts directory and take a look at some of the python files in the examples\python directory and in the tests directory.
